frontend/src/components/common/link/link.tsx

```diff
@@ -4,7 +4,7 @@ import { AppRoute } from 'common/enums/enums';
 
 type Props = {
   to: AppRoute;
-  className?: string;
+  className?: string | ((props: { isActive: boolean }) => string);
```

Comment:
Better to send needed classname from wrap component. Maybe I missed something, add comments if so

Reply:
But in general it does not look correct